### DISCUSSION TITLE
Is there a way to generate a smart response on a different interface?

### DISCUSSION THREAD

**Discussion by fgolman** [dannagle/PacketSender#339] [Q&A]
I'd like to monitor on UDP and then send out response via HTTP Post, for example?

**Comment by dannagle** [comment]
This idea is similar to the proxy and forwarding service suggested by #75 and #338 .  Tying that into a smart response is an interesting idea.

There is no direct way to do this. There are a lot of good ideas surrounded by smart responses. I may need to break backward compatibility of previously saved smart responses to be able to implement some of them.
